app/models/miq_worker_type.rb

```diff
@@ -0,0 +1,24 @@
+class MiqWorkerType < ApplicationRecord
+  EXCLUDED_CLASS_NAMES = %w[ManageIQ::Providers::BaseManager::OperationsWorker].freeze
+
+  scope :in_kill_order, -> { order(:kill_priority => :asc) }
+
+  def self.seed
+    transaction do
+      classes_for_seed.each { |klass| seed_worker(klass) }
```

Comment:
Will this check for workers which are in the table but not longer exist in the `classes_for_seed` ? aka if we delete a worker

Reply:
Ah, good catch. Nope.